lab-geoff/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "lab-07-cowsay-server",
+  "version": "1.0.0",
+  "description": "Assignment is to create an HTTP Server and a parse body module that is used for POST requests.",
+  "main": "server.js",
+  "scripts": {
+    "test": "mocha"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/geoffreycc/lab-07-cowsay-server.git"
+  },
+  "author": "geoff-cox",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/geoffreycc/lab-07-cowsay-server/issues"
+  },
+  "homepage": "https://github.com/geoffreycc/lab-07-cowsay-server#readme",
+  "devDependencies": {
+    "cowsay": "^1.1.8",
```

Comment:
Cowsay should not be in the devDeps, given that it's a requirement for the app to function.